README.md

```diff
@@ -297,6 +295,57 @@ make build-linux-amd64
 make build-all
 ```
 
+### Node-locking
+
+You may node-lock Relay to a specific machine, but doing so requires custom
```

Comment:
Wording here is awkward. It's a great way to facilitate deploying Relay in untrusted environments.